src/components/InputForm/index.js

```diff
@@ -4,25 +4,27 @@ import classes from './style.module.css';
 
 class InputForm extends Component {
   state = {
-    content: ''
+    content: '',
   }
 
-  changeHandler = event => {
-    this.setState({content: event.target.value});
-  }
+  changeHandler = (event) => {
```

Comment:
This can be written as `changeHandler = event => this,setState({ content: event.target.value });` on one line. Unless, it goes past the max line length.

Reply:
Nope. Sounds good.